js/indicia.datacomponents/idc.core.js

```diff
@@ -752,6 +752,24 @@
       return Math.abs(lon).toFixed(3) + (lon >= 0 ? 'E' : 'W');
     },
 
+     /**
```

Comment:
These will need to be merged with the existing lat and lon functions using a format option.

Reply:
Oh yes - I will take a look.

Reply:
@johnvanbreda - I'm trying to work out where to test these JS field handlers - where do they kick in?

Reply:
Create a [download] pointing to a [source], then set columnsTemplate to a blank string and use addColumns to add a #lat# column. E.g. (untested)
```
[source]
@id=data-to-download

[download]
@source=data-to-download
@columnsTemplate=
@addColumns=<!--[
  {"caption": "Lat", "field": "#lat#},
  {"caption": "Lon", "field": "#lon#}
]-->
```